web/src/features/exchanges/ExchangeArrow.tsx

```diff
@@ -108,29 +108,26 @@ function ExchangeArrow({
   }
 
   const mapZoom = map.getZoom();
-  const projection = map.project(lonlat);
-  const transform = {
-    x: projection.x,
-    y: projection.y,
-    k: 0.04 + (mapZoom - 1.5) * 0.1,
-    r: rotation + (netFlow > 0 ? 180 : 0),
-  };
+  const { x, y } = map.project(lonlat);
+  const k = 0.04 + (mapZoom - 1.5) * 0.1;
+  const r = rotation + (netFlow > 0 ? 180 : 0);
+  const factor = 100 * k;
```

Comment:
Could do some nice naming here:

```suggestion
  const baseZoom = 0.04 + (mapZoom - 1.5) * 0.1;
  const directionalRotation = rotation + (netFlow > 0 ? 180 : 0);
  const scaledZoom = 100 * k;
```

Would also need to update below

Reply:
I originally kept the names the same so the diff would be as small as possible but I'd be happy to rename them before merging!

Reply:
Sounds good!